### PR TITLE
Interpreter properties editor updates

### PR DESCRIPTION
**Pull request by r-kamath**
- Add extra validations in add new interpreter property function to
  prevent 'undefined'
- Instead of an empty table show a message when there are no properties to display
- Apply bootstrap style for the table

Before
screen shot 2015-12-09 at 2 34 48 pm
screen shot 2015-12-09 at 2 34 36 pm
<hr>
After
screen shot 2015-12-09 at 2 36 07 pm
screen shot 2015-12-09 at 2 35 47 pm

### CODE REVIEW

File: zeppelin-web/src/app/interpreter/interpreter.html [hunk 1]
```diff
@@ -67,15 +67,20 @@ <h3 class="interpreter-title">{{setting.name}}
     </div>
     <br />
     <div class="row interpreter">
-      <div class="col-md-12">
-        <b>Properties</b>
+      <div ng-show="getInterpreterPropertiesCount(setting.id) === 0 || valueform.$hidden" class="col-md-12 empty-properties-message">
```
Comment (corneadoug) [inline zeppelin-web/src/app/interpreter/interpreter.html]: ``getInterpreterPropertiesCount(setting.id) === 0``
can be replaced by ``_.isEmpty(setting.properties)`` if you add ``$scope._ = _;`` in the controller.

File: zeppelin-web/src/app/interpreter/interpreter.html [hunk 2]
```diff
@@ -67,15 +67,20 @@ <h3 class="interpreter-title">{{setting.name}}
     </div>
     <br />
```
Comment (corneadoug) [inline zeppelin-web/src/app/interpreter/interpreter.html]: this <br /> is adding too much space, we could remove it

**Comment by corneadoug** [comment]
I don't like the new table style, because I think that grey background doesn't go well with the rest.
Originally the header of the table component is even supposed to be white. However I like the way it was as a striped row because it is similar to the graph table header background color. To me, having the text in bold is enough to know its a header.

**Comment by r-kamath** [comment]
@corneadoug thanks for the review. Removed the table header style and other changes in  de75e49

**Comment by corneadoug** [comment]
By adding this to interpreter.css:
```
table.table-striped {
    border-top: 1px solid #ddd;
    margin-top: 20px;
}
```
We are making Table Header stand out a bit more, and it looks less crowded

screen shot 2015-12-16 at 11 23 59 am

Otherwise LGTM

**Comment by corneadoug** [comment]
Build fail is not related (random timeout fail), you can just re-trigger CI and it should be green

**Comment by r-kamath** [comment]
@corneadoug border-top looks good

**Comment by corneadoug** [comment]
Merging If There is no more discussions

**Comment by prabhjyotsingh** [comment]
LGTM

**Pull request closed without merging**